config.toml

```diff
@@ -127,7 +127,7 @@ max_msg_num = 30
 max_tx_size = 2097152
 clock_drift = "5s"
 max_block_time = "30s"
-trusting_period = "75s"
+trusting_period = "24h"
```

Comment:
It seems we can also make this change in `starship.yml` using the their Directive Syntax: 

https://github.com/cosmology-tech/starship/blob/e399ad315ce1612221dc70f7b6807c4af7b4d94e/charts/devnet/templates/relayers/hermes/configmap.yaml#L127